litmus-portal/graphql-server/graph/schema.resolvers.go

```diff
@@ -63,22 +66,52 @@ func (r *mutationResolver) DeleteChaosWorkflow(ctx context.Context, workflowid s
 }
 
 func (r *mutationResolver) SendInvitation(ctx context.Context, member model.MemberInput) (*model.Member, error) {
+	err := validate.ValidateRole(ctx, member.ProjectID, requiredRoles[2:3], []string{"Accepted"})
+
+	if err != nil {
+		return nil, err
+	}
+
 	return project.SendInvitation(ctx, member)
 }
 
 func (r *mutationResolver) AcceptInvitation(ctx context.Context, member model.MemberInput) (string, error) {
+	err := validate.ValidateRole(ctx, member.ProjectID, requiredRoles[:2], []string{"Pending"})
+
+	if err != nil {
+		return "Unsuccessful", err
+	}
+
 	return project.AcceptInvitation(ctx, member)
 }
 
 func (r *mutationResolver) DeclineInvitation(ctx context.Context, member model.MemberInput) (string, error) {
+	err := validate.ValidateRole(ctx, member.ProjectID, requiredRoles[:2], []string{"Pending"})
+
+	if err != nil {
+		return "Unsuccessful", err
+	}
+
 	return project.DeclineInvitation(ctx, member)
 }
 
 func (r *mutationResolver) RemoveInvitation(ctx context.Context, member model.MemberInput) (string, error) {
+	err := validate.ValidateRole(ctx, member.ProjectID, requiredRoles[2:3], []string{"Accepted", "Pending"})
```

Comment:
Invitation status should be just "Accepted"?

Reply:
Yes, this should be just "Accepted", @SarthakJain26 is changing it.